chat-client/.npmignore

```diff
@@ -1,4 +1,7 @@
+build/
 node_modules/
+src/
 package-lock.json
 .*
-tsconfig.*
\ No newline at end of file
+tsconfig.*
+webpack.*.config.js
```

Comment:
is the `*` intentional? this means we are publishing `webpack.*.config.js`

Reply:
it's intentional to prevent publishing all possible webpack.config files. Same rule works with `codewhisperer` package https://www.npmjs.com/package/@aws/lsp-codewhisperer?activeTab=code fine

Reply:
it prevents possible configs but not the actual config which is `webpack.config.js` (this is what I tried to say in my first comment but mistakenly put the star in the middle, updated my initial comment as well)

So we should either put: `webpack.*` or `*.config.js` here.  The reason it works in the other subpackage is because we have `webpack.lint.config.js` over there, in chat client we have  `webpack.config.js`

Reply:
yeah, you are right, fixed it